src/mock/handlers/rating.ts

```diff
@@ -0,0 +1,7 @@
+import { rest } from 'msw';
+
+export default [
+    rest.put('/api/townhalls/:townhallId/rating', (req, res, ctx) => {
```

Comment:
I made some requested changes on the backend, you'll need to change this to a post once you change the backend.